vcf-dumper/vcf-dumper-lib/src/test/java/uk/ac/ebi/eva/vcfdump/BiodataVariantToVariantContextConverterTest.java

```diff
@@ -77,7 +78,7 @@ public void singleStudySNV() {
         String variantLine = String
                 .join("\t", CHR_1, "1000", "id", "C", "A", "100", "PASS", ".", "GT", "0|0", "0|0", "0|1", "1|1", "1|1",
                       "0|1");
-        List<Variant> variants = variantFactory.create(variantSource, variantLine);
```

Comment:
"variantLine" objects are not being used anymore, because the variants are being created now using VariantWithSamplesAndAnnotation constructor. 

Also, all the variants used in the test are now the same: Chromosome "c", start 1, end 10, ref "ref and alternate null. Please create a suitable variant for each test.

Reply:
variantLine objects are used again with the new factories